src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
+            height: 100%;
+        }
+
+        .be {
+            height: 100%;
+
+            .bcs-activity-feed {
+                width: 100%;
+            }
+        }
+
+        .bcs-activity-feed-comment-input {
+            width: 100%;
+        }
+
+        .be.bcs {
+            flex-direction: column;
+            width: 100%;
+            min-width: unset;
+            max-width: unset;
+            height: 100%;
+            max-height: 48px;
+            -webkit-transition: max-height .5s ease-in-out 0s;
```

Comment:
We should have autoprefixer, making this unnecessary.

Reply:
Understood. Going to remove vendor prefixes.